docs/cloud-providers-credentials/generate-gcp-service-account-credentials.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Generating Google Service Account Credentials
```

Comment:
in this documentation I am using the verb "generate" to describe how to get the credentials of a GCP service account in Google Cloud Console, it doesn't seem to make sense to use something like "Add", since the documentation to actually add the credential in Devopness can be found in https://github.com/devopness/devopness/blob/main/docs/cloud-providers/add-cloud-providers-credentials.md

Reply:
Using `get` so it represents the the process to obtain GCP service account credentials

https://github.com/devopness/devopness/blob/614e3070a24c883a787e519c4821679fe2bbfc67/docs/cloud-provider-credentials/get-gcp-service-account-credentials.md?plain=1#L2